### PR TITLE
Add tree inventory data QA/QC Script

### LINKED ISSUE #20: Tree inventory QA/QC analysis script
Hi @10aDing thanks for being willing to work on this!

The `inventory.csv` file describes, for each tree in the TEMPEST plots, its "DBH" (diameter) in 2019 and 2020 (and soon 2021):

Plot | Section | Tag | DBH_2019 | Date_2019 | Species_code | Status_2019 | Sapflux | DBH_2020 | Date_2020 | Status_2020 | Notes | JS_codes
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Control | East | 1301 | 10.7 | 3/9/19 | COFL2 | Yes |   | 10.7 | 3/10/20 |   |   |  
Control | East | 1302 | 11.7 | 3/9/19 | ACRU | Yes |   | 11.7 | 3/10/20 |   |   |  
Control | East | 1303 | 18.7 | 3/9/19 | ACRU | Yes |   | 18.9 | 3/10/20 |   |   |  
Control | East | 1304 | 51.8 | 3/9/19 | LIST2 | Yes |   | 52.2 | 3/10/20 |   |   |  

We would like a script that does qa/qc (error checking and visualization), along later with some computation on the data. Initially:
* Create a new branch for your work
* Create a new R Markdown file in the `R_scripts/` folder 
* Read in the above dataset
* Produce a summary table (probably using `dplyr` functions, but up to you) showing, by plot and species, number of trees 
* Identify any trees whose diameter changed more than ±0.5 cm between 2019 and 2020
* Make a nice graph of the data! Think about how we might visualize the above information

### PR DESCRIPTION
**Pull request by 10aDing**
Add the QA/QC script specified in Issue #20 to create a long table, graph data, and identify errors.

Closes #20

### CODE REVIEW

**Comment by bpbond** [comment]
Hello @ahopple @stephpenn1 we are hoping you have time to provide feedback to Colin in the next week?

**Comment by stephpenn1** [comment]
Really nice work @10aDing. Is there somewhere I can see the document?

**Comment by bpbond** [comment]
@stephpenn1 @ahopple I have posted the output from Colin's script here: https://rpubs.com/bpbond/767657

Will keep it updated with any changes/feedback from you.

File: R_scripts/Colin_QAQC.Rmd [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+scatterpie_dist <- ggplot() + geom_scatterpie(aes(X, Y, r = total/18), data = scatter_inventory, cols = colnames(scatter_inventory)[-(1:4)]) + coord_equal() + facet_wrap(~Plot, ncol=1) + scale_x_continuous(breaks = 1:10, labels=c(LETTERS[1:10])) + xlab(NULL)
+
+scatterpie_dist
+```
+
+# Error Check {.tabset}
+## Zombie Trees
+```{r}
+zombie_trees <- long_inventory %>%
```
Comment (stephpenn1) [inline R_scripts/Colin_QAQC.Rmd:124]: @10aDing a comment here on what you are doing would be helpful

File: R_scripts/Colin_QAQC.Rmd [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+```{r}
+wrong_status <- long_inventory %>%
+  group_by(Tag) %>%
+  filter(!all(sapply(grep(NA, Status), function(x, y) all(x < y), grep(!NA, Status))) | (!grepl("LI|D[SC]", Status) & !is.na(Status))) %>% 
+  select(Tag, Year, Status, Notes)
+wrong_status
+```
+
+## Incorrect DBH
+```{r}
```
Comment (stephpenn1) [inline R_scripts/Colin_QAQC.Rmd:148]: And a comment here too explaining why you chose 1 as the cutoff

**Comment by stephpenn1** [comment]
@10aDing I have two comments on your code which you can view in the `Files Changed` tab about places where some comments would be useful, otherwise it looks great.

**Comment by bpbond** [comment]
@10aDing Will you have a chance to address Steph's comments? Thanks

**Comment by 10aDing** [comment]
@bpbond Sorry about that! I'll take a look at the comments ASAP

File: R_scripts/Colin_QAQC.Rmd (hunk 1 above)
Comment (stephpenn1) [reply R_scripts/Colin_QAQC.Rmd:126]: Nice job!

**Comment by bpbond** [comment]
@stephpenn1 I think @ahopple has had plenty of time to weigh in; feel free to merge  :)

**Comment by bpbond** [comment]
I guess I am merging. Thank you @10aDing — great job on this!

**Pull request merged**